.travis.yml

```diff
@@ -23,5 +23,5 @@ script: ./tool/travis.sh
 # Testing product matrix; the values are generated from product-matrix.json.
 env:
   - DART_BOT=true
-  - IDEA_VERSION=2017.3
-  - IDEA_VERSION=2018.1
+  - CI=true IDEA_VERSION=2017.3
+  - CI=true IDEA_VERSION=2018.1
```

Comment:
Curious what `CI=true` does?

Reply:
CI identifies the run as a travis CI run to the plugin tool. It is used during the build to auto-edit some files on travis, so no one has to hack the plugin.dart code to make a local build work.

Reply:
Cool.  Thanks for the info!